src/components/AmountTextInput.js

```diff
@@ -7,14 +7,31 @@
 
 import React from 'react';
 import { StyleSheet, TextInput } from 'react-native';
+import { bigIntCoercibleSchema } from '@hathor/wallet-lib/lib/utils/bigint';
```

Comment:
Remove the import? The check identified it's not in use.

Reply:
Done